src/interchange/benches/avro.rs

```diff
@@ -393,7 +393,9 @@ pub fn bench_avro(c: &mut Criterion) {
     buf.extend(mz_avro::to_avro_datum(&schema, record).unwrap());
     let len = u64::cast_from(buf.len());
 
-    let mut decoder = Decoder::new(schema_str, None, "avro_bench".to_string(), false).unwrap();
+    let mut decoder =
+        Decoder::<Box<mz_ccsr::Client>>::new(schema_str, None, "avro_bench".to_string(), false)
```

Comment:
Is there no blanket impl for `T -> &T` for `Deref`?

Reply:
there is not :(

we can switch this to `Borrow` in the future if it seems to annoying, but the `Deref` impl is the minimal change